### ISSUE TITLE
Feature Request: Accounts for parents?

### ISSUE THREAD

**Issue by white-rabbit07** [linuxmuster/sophomorix4#155]
Hi.
it would be nice to have accounts for parents like this:

Pupils account: 
Name                          Login           Group
max mustermann ---> musmax --> Klasse 5a 

Parents: 
e.musmax --> Eltern 5a (and the same default-password for everyone??)

Is there a way to realize that? The background of this question is: 
-  A way for parents to login via LDAP/AD  to moodle (and other platforms) with correct permissions and allocations.
- User-Management on **one** Server for everyone

**Comment by white-rabbit07** [comment]
> Just one small note: it is necessary to provide two accounts for divorced parents.

I don't think so ... divorced parents should talk with each other and use their account together. This can't be a problem of the school server. But there's a problem with brothers and sisters ... parents could get 3 or 4 accounts this way which is no good solution for this problem?!?

**Comment by dorianim** [comment]
> divorced parents should talk with each other and use their account together

This does not work in all cases. I can ensure that from my own experience. There must be two accounts for the parents, at least optionally.

**Comment by PLanB2008** [comment]
Providing logins for parents should be discussed and designed in detail. Im not a fan of these "quick fire and forget solutions".
* Default Passwords are always a bad idea
* Using an e for the naming scheme "eltern" would break the English first naming scheme
* Using a dot to separate users is not supported in linuxmuster for fallback cases in linux

We need to look into the LDAP design. In my opinion such a feature would need its own OU. In addition we would need to handle the obvious m:n relation between parents (mum and dad as suggested by @dorianim ) and 1 to n children. 

I won't say this is a bad idea by any means but there are even more aspects to keep in mind.

I would suggest scheduling this topic for a developer meeting.
